apis/accounts/slashing_protection.yaml

```diff
@@ -0,0 +1,70 @@
+get:
+  tags:
+    - SlashingProtection
+  operationId: "ExportSlashingProtection"
+  summary: "Export slashing protection data."
+  description: |
+    MUST return a file with the format defined in EIP-3076: Slashing Protection Interchange Format.
+    If pubkeys is not specified MUST return data for all pubkeys in the local slashing protection DB.
```

Comment:
Exporting the slashing protection data is futile if the validator client continues to sign messages with the keys after the export. The exported data will become stale almost immediately and offer no protection.

I think any API that exports slashing protection data should also atomically disable the keys for which the data was exported. We could support this either by keeping the API as-is and specifying that the exported keys will also be disabled, or by creating a new unified `/export` endpoint that:

1. Exports the keystores for the requested validator public keys (optionally all of them), including passwords and
2. Exports the slashing protection data for the requested validators

whilst

3. Disabling all requested validator public keys so that no new messages are signed which aren't included in the slashing data from (2)

We could create nice symmetry by allowing the API for _importing_ keystores to accept optional slashing protection data, so that you can feed the output of the export endpoint into the import endpoint in order to migrate validators from one VC to another (we've been discussing this for Lighthouse here: https://github.com/sigp/lighthouse/issues/2557).

If there are no objections to the above design I could try speccing it as a PR to this PR.